test/Microsoft.IdentityModel.Logging.Tests/LoggerTests.cs

```diff
@@ -265,5 +267,67 @@ public void EventLevelToEventLogLevelMapping()
             Assert.True(logger.ContainsLogOfSpecificLevel(string.Format(verboseMessage, arg), EventLogLevel.Verbose));
             Assert.True(logger.ContainsLogOfSpecificLevel(string.Format(warnMessage, arg), EventLogLevel.Warning));
         }
+
+        [Theory, MemberData(nameof(LoggerTestTheoryData))]
+        public void LoggerInstanceTests(LoggerTheoryData theoryData)
+        {
+            LogHelper.Logger = theoryData.Logger;
+
+            if (theoryData.Logger != null)
+            {
+                Assert.True(theoryData.ShouldMessageBeLogged == LogHelper.Logger.IsEnabled(theoryData.EventLogLevel));
+            }
+        }
+
+        public static TheoryData<LoggerTheoryData> LoggerTestTheoryData
+        {
+            get
+            {
+                var theoryData = new TheoryData<LoggerTheoryData>();
+
+                theoryData.Add(new LoggerTheoryData
+                {
+                    TestId = "NullLoggerNoMessage",
+                    Logger = null,
+                    ShouldMessageBeLogged = false
+                });
```

Comment:
Is this test actually doing anything? It just skips over the if statement and passes correct? Seems like this only exercises test code.

Reply:
True, it doesn't add a value. Removed it from the test data. It was left there to show that no message can be written if logger is set to null.